ghost/core/test/e2e-api/admin/posts-legacy.test.js

```
@@ -1472,6 +1472,41 @@ describe('Posts API', function () {
+        updatedPost.email_only = true;
+        updatedPost.published_at = moment().add(2, 'days').toDate();
+
+        // Attempt to schedule without newsletter reference should fail
+        await request
+            .put(localUtils.API.getApiQuery('posts/' + id + '/'))
+            .set('Origin', config.get('url'))
+            .send({posts: [updatedPost]})
+            .expect('Content-Type', /json/)
+            .expect('Cache-Control', testUtils.cacheRules.private)
+            .expect(422)
+            .expect((res) => {
```

ESLint

[error] 'res' is already declared in the upper scope on line 1482 column 15.

(no-shadow)